### ISSUE TITLE
mariadb (azuracast_db) container won't start: Error while loading shared libraries: libtinfo.so No such file or directory

### ISSUE THREAD

**Issue by emmanuelproulx** [AzuraCast/AzuraCast#2734]
**Using Docker installation method**
Yes

**Host Operating System**
CentOS 8 running on Google Cloud VM.

**Describe the bug**
I've installed Azuracast with Docker less than 2 weeks ago.

I must say that I'm quite surprised this stopped working. It was working very well. All I did was reboot the host after adding a user to a Linux group.

After restarting, two docker containers stopped working. One of them is mariadb (image azuracast/azuracast_db:latest). 

Google gave me almost nothing to work with. Someone was mentioning a missing symbolic link `/usr/lib/libtinfo.so` which should point to (I guess) `/lib/x86_64-linux-gnu/libtinfo.so.5`. So I bash into the instance and see that the link is not there. When I add this to the docker-compose script: 
`    command: ln -s /lib/x86_64-linux-gnu/libtinfo.so.5 /usr/lib/libtinfo.so`
this gives me an error when I run: `failed to create symbolic link '/usr/lib/libtinfo.so': File exists `

Anyway, it doesn't explain why it used to work and not it doesn't.

**To Reproduce**
1. Install Azuracast with Docker.
2. Reboot the host with `sudo shutdown -r 0`
3. Look at the logs for the mariadb container

Alternatively, you can run the service manually with:
`sudo docker-compose up mariadb`

**Expected behavior**
The mariadb container should start without error.

**Relevant Logs**
```
mariadb_1   | 2020/04/24 19:45:52 Command exited with error: exit status 127
mariadb_1   | 2020-04-24 19:46:54+00:00 [Note] [Entrypoint]: Entrypoint script for MySQL Server 1:10.4.12+maria~bionic started.
mariadb_1   | 2020-04-24 19:46:54+00:00 [Note] [Entrypoint]: Switching to dedicated user 'mysql'
mariadb_1   | /bin/bash: error while loading shared libraries: libtinfo.so.5: cannot open shared object file: No such file or directory
```

**Screenshots**
None

**Device(s):**
Irrelevant

**Additional context**
I noticed that the Docker image did not change for a long time so this isn't due to a recent change. I also tried deleting the image and recreating it. I'm at a loss wrt the cause of the error.

**Comment by emmanuelproulx** [comment]
I added this command to docker-compose.yml:
`    command: ln -s /lib/x86_64-linux-gnu/libtinfo.so.5 /usr/lib/libtinfo.so.5`

and the error went away. Instead there's an error about unable to create this softlink because it already exists:

```
mariadb_1   | ln: failed to create symbolic link '/usr/lib/libtinfo.so.5': File exists
mariadb_1   | 2020/04/25 13:59:45 Command exited with error: exit status 1
```
But then, the container is stuck in the restarting state. No further error.

I'm starting to think that these log errors are not actually related to the root cause of the container not starting.

**Comment by CodeSteele** [comment]
Likely a duplicate of #2744 -- go ahead and remove/recreate the container, if this fixes the other issue but not this one feel free to reopen.

**Comment by emmanuelproulx** [comment]
I tried that and it did nothing.

**Comment by emmanuelproulx** [comment]
Also, why would it be a duplicate if the error message is completely different?

**Comment by CodeSteele** [comment]
because they're both pretty much 3rd party base images that we don't really touch the base image on, and both exhibit what could be best described as "possible corruption of the underlying docker images".

**Comment by emmanuelproulx** [comment]
Finally, the issue #2744 is fixed. So this issue was not a duplicate of it.

**Comment by emmanuelproulx** [comment]
In order to debug this issue, I built the database component image locally. Then I started `docker-compose up -d`. Mysteriously, with this local image, the error is gone. It doesn't explain the cause of the issue but it's a workaround.

**Comment by CodeSteele** [comment]
Though this still sounds like a duplicate of #2744 where something is _terribly_ wrong with your docker images, your host, your storage, etc. because these files are parts of images we do not manage, the other option is to try running the MariaDB image we use from the docker command line and seeing if that errors. We run with `mariadb:10.4-bionic` so you can just try firing that up and see if that gives you the lib error.

**Comment by emmanuelproulx** [comment]
I'm unblocked but still not sure why it didn't work. All I did to make it work is change docker-compose.yml and change the mariadb image:
image: azuracast_db_dev
this is my own docker build I did on the server directly and I tagged it _dev. I didn't change anything else; volumes, Dockerfile, nothing.

I will no longer pursue this unless you want to have a look yourself.

**Comment by CodeSteele** [comment]
The only concern I have is if we update our image to use a newer version of MariaDB and new features on it, but other than that, I'm pretty sure it's a host-related issue so it's not really something I'd rather poke at because it's way outside of the scope of supporting our software.

**Comment by emmanuelproulx** [comment]
If that happens I will have to get the latest. Thanks for your time.
